dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/export/openaire/CleanupTest.java

```diff
@@ -0,0 +1,28 @@
+package edu.harvard.iq.dataverse.export.openaire;
+
+import org.junit.Assert;
+import org.testng.annotations.Test;
+
+/**
+ *
+ * @author francesco.cadili@4science.it
+ */
+public class CleanupTest {
```

Comment:
We should keep tests separate from non tests code:
`dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/export/openaire/CleanupTest.java` -> `dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/export/openaire/CleanupTest.java`